examples/niscope/graph-measurement-betterproto.py

```diff
@@ -119,7 +119,7 @@ async def ConfigureGrpcScope(scope_service: niscope_grpc.NiScopeStub, channel, v
     set_trigger_result = await scope_service.set_attribute_vi_int32(
         vi = vi,
         attribute_id = niscope_grpc.NiScopeAttributes.NISCOPE_ATTRIBUTE_TRIGGER_TYPE,
-        value = niscope_grpc.NiScopeInt32AttributeValues.NISCOPE_INT32_TRIGGER_TYPE_VAL_EDGE_TRIGGER
+        value_raw = niscope_grpc.NiScopeInt32AttributeValues.NISCOPE_INT32_TRIGGER_TYPE_VAL_EDGE_TRIGGER
```

Comment:
Can you add this comment here so that customers are aware issue with betterproto in gRPC and why we are setting `value_raw` here.

> We have to set `value_raw` instead of `value` here because of `betterproto` bug related to `oneof` types in gRPC messages. More information here: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client